### PR TITLE
Added recipe for fixtures

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/fixtures/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  license: Apache 2.0 or BSD 3-Clause
+  license_file: COPYING
+  summary: 'Fixtures, reusable state for writing clean tests and more.'
+  dev_url: https://github.com/testing-cabal/fixtures
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
+
+package:
```
Comment (ocefpaf) [inline recipes/fixtures/meta.yaml]: I believe these line here are left-overs, right?

File: recipes/fixtures/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - fixtures
+    - fixtures.callmany
+    - fixtures.fixture
+    - fixtures.testcase
+
+
+about:
+  home: https://github.com/testing-cabal/fixtures
+  license: Apache 2.0 or BSD 3-Clause
+  license_file: COPYING
```
Comment (ocefpaf) [inline recipes/fixtures/meta.yaml]: Which one did you choose? :wink:

Comment (pmlandwehr) [inline recipes/fixtures/meta.yaml]: Hah. `COPYING` rolls both in fortunately, so I didn't.

Comment (jakirkham) [inline recipes/fixtures/meta.yaml]: Sort of. Seems to include a reference to both without the licenses per se.

That being said, this case feels like a `conda-build` issue IMHO as there is not a manner of specifying two licenses for cases just like this. Please see issue ( https://github.com/conda/conda-build/issues/1247 ).

File: recipes/fixtures/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  summary: 'Fixtures, reusable state for writing clean tests and more.'
+  dev_url: https://github.com/testing-cabal/fixtures
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/fixtures/meta.yaml]: Any interest in helping to maintain @anguslees?

**Pull request merged**